### PR TITLE
Redo of PR #2190: aggregated logging file edits

### PR DESCRIPTION
**Pull request by bfallonf**
A second PR cos I stuffed up the squash on the first.

@tpoitras @ahardin-rh @adellape @tnguyen-rh Probably best to make any comments here. Thanks.

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -42,22 +44,11 @@ to manage container logs and prevent filling node disks.
...
-* A project, typically reserved just for this deployment
-* A *logging-deployer* secret
-* A *logging-deployer* service account with access to the secret and creation privileges
-
-Perform the following steps to prepare for deployment:
-
-. Ensure that you have deployed a router for the cluster.
-. Ensure that you have the necessary storage for Elasticsearch.
+. Ensure that you have xref:../install_config/install/deploy_router.html#[deployed a router] for the cluster.
+. Ensure that you have xref:../install_config/persistent_storage/index.html[the
```
Comment (tnguyen-rh) [inline install_config/aggregate_logging.adoc]: s/.html/.adoc/  (twice)

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -96,14 +87,13 @@ $ oc project logging
 [NOTE]
 ====
 Specifying a non-empty
-link:../admin_guide/managing_projects.html#using-node-selectors[node
+xref:../admin_guide/managing_projects.html#using-node-selectors[node
```
Comment (tnguyen-rh) [inline install_config/aggregate_logging.adoc]: s/html/adoc/

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -96,14 +87,13 @@ $ oc project logging
...
 selector] on the project is not recommended, as this would restrict
 where Fluentd can be deployed. Instead, specify node selectors for the
-deployer to be applied on your other deployment configurations.
+deployer to be applied to your other deployment configurations.
 ====
 
-. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret
-can be configured specifically for your instance of the deployer.
+. Create a xref:../dev_guide/secrets.html#[secret] to provide security-related files to the deployer. While the secret is necessary, the contents of the secret are optional, and will be generated for you if none are supplied.
```
Comment (tnguyen-rh) [inline install_config/aggregate_logging.adoc]: s/html/adoc/

**Comment by tnguyen-rh** [comment]
@bfallonf A few instances of `xref:` with `.html` to fix, otherwise LGTM.

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -12,14 +12,15 @@ toc::[]
 
 == Overview
 
-OpenShift cluster administrators can aggregate logs and enable application
-developers to view them for a range of OpenShift services using the EFK stack,
-which is a modified version of the
-https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. The EFK
-stack can be useful for viewing logs aggregated from hosts and applications,
-whether coming from multiple containers or even deleted pods.
+As an OpenShift cluster administrator, you can deploy the EFK stack to aggregate
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: Replace Openshift with {product-title}? I see 2 instances here.

File: install_config/aggregate_logging.adoc [hunk 5]
```diff
@@ -398,28 +389,32 @@ endif::openshift-enterprise[]
...
 
-In any production employment, Elasticsearch should be deployed with a
-cluster size of at least three for resiliency to node failures. Each instance
-requires individual storage, but an OpenShift deployment can only provide
-volumes shared by all its pods, so the Elasticsearch cluster cannot be
-implemented with a single deployment. The EFK deployer instead creates one
-deployment per instance. To view all current deployments of Elasticsearch:
+A highly-available environment requires at least three replicas of
+Elasticsearch; each on a different host. Elasticsearch replicas require their
+own storage, but an OpenShift deployment configuration shares storage volumes
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: s/OpenShift/{product-title}

**Comment by ahardin-rh** [comment]
@bfallonf I just found a few instances where "OpenShift" should probably be replaced with {product-title}. Other than that LGTM!

**Comment by bfallonf** [comment]
@ahardin-rh @tnguyen-rh Thanks. Changes done. Both cases are because I wasn't paying better attention to little obvious things... I'll merge away.

**Comment by bfallonf** [comment]
[rev_history]
|link:../install_config/aggregate_logging.html[Aggregating Container Logs]
|Rewording and clarifications.
%

**Pull request merged**